actions/views/rhs.ts

```diff
@@ -287,13 +287,18 @@ export function showPinnedPosts(channelId?: string) {
 
 export function showChannelFiles(channelId: string) {
     return async (dispatch: (action: Action, getState?: GetStateFunc | null) => Promise<ActionResult|[ActionResult, ActionResult]>, getState: GetStateFunc) => {
-        const state = getState();
+        const state = getState() as GlobalState;
```

Comment:
Nit: is this type assertion needed?

Reply:
Yep, because we have both `mattermost-redux/types/store/GlobalState` and `types/store/GlobalStore` co-existing